node-process-argv/process.js

```diff
@@ -0,0 +1,3 @@
+
+const process = require('process');
```

Comment:
`process` is globally available so you don't need to `require` it.

Reply:
got it!